clients/da-vinci-client/src/test/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTaskTest.java

```diff
@@ -3084,6 +3085,7 @@ public void testLeaderShouldSubscribeToCorrectVTOffset() {
         .getLocalStorageEngine(anyString());
     doReturn(mockStorageEngineRepository).when(builder).getStorageEngineRepository();
     VeniceServerConfig veniceServerConfig = mock(VeniceServerConfig.class);
+    doReturn(new VeniceProperties(new Properties())).when(veniceServerConfig).getClusterProperties();
```

Comment:
Instead of `new VeniceProperties(new Properties())`, let's use `VeniceProperties.empty()`